charters/MetadataSchema/charter.md

```diff
@@ -0,0 +1,63 @@
+
+# Metadata Schema
+
+## Description
+
+The HCA Metadata schema committee creates and maintains the Human Cell Atlas metadata schema in JSON format. These schema are designed to capture and provide structure for the descriptive scientific metadata associated with HCA datasets. These schemas aim to ensure the FAIRness* of the HCA data [The FAIR Guiding Principles for scientific data management and stewardship](https://www.nature.com/articles/sdata201618). 
+
+## Objective
```

Comment:
I'd like to see a definition of how comprehensive metadata schemas are expected to be in order to deliver on scientific aims. I think it is implicit here that schemas will aim to represent the contents of data submitted to the HCA - and it is worth making this explicit - but I would also like to see a declaration about whether the metadata team aim to produce a minimum information style standard, a fully comprehensive one, or something in between (and if something in between, what?)

Reply:
I have added another paragraph to the description which addresses this. Do let me know what you think.